src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  --blue-color: #00acdc;
+  --main-color: #000;
+}
+
+.nav__link {
+  text-decoration: none;
+  display: block;
+  padding-top: 22.8px;
+  padding-bottom: 22.8px;
+  color: var(--main-color);
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__list {
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin: 0 10px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  flex-shrink: 0;
+  margin: 10px 0;
+}
+
+.nav__list :hover {
+  color: var(--blue-color);
+}
+
+a.is-active {
+  position: relative;
+  color: var(--blue-color);
+  width: 100%;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  bottom: 0;
+  background-color: var(--blue-color);
+  left: 0;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)